Tools/TestWebKitAPI/Tests/WebCore/LayoutUnitTests.cpp

```diff
@@ -36,6 +36,10 @@ using namespace WebCore;
 
 namespace TestWebKitAPI {
 
+namespace {
+    constexpr float tolerance = 1.0f / kFixedPointDenominator;
```

Comment:
You can drop the indent, so that we comply with https://webkit.org/code-style-guidelines/#indentation-namespace

Reply:
Thanks! For some reason style checker didn't complain. Surprised!

Reply:
[Done](https://github.com/WebKit/WebKit/commit/78719f76a3660f1353db9a27d4a1ca123edefeba).